contents/tutorials/when-and-how-to-run-group-targeted-a-b-tests.md

```diff
@@ -0,0 +1,88 @@
+
+## Combining user-targeted and group-targeted experiments to reduce risk
+
+To improve the quality of insights from group-targeted experiments and address the risks mentioned above, you may want to consider running an experiment with your change at a user-level before running it at a group-level. This can help you in the following ways:
+
+### 1. Save time for changes that don't have an impact. 
+
+As mentioned, lower statistical power in group-targeted experiments means you usually need to run your experiments longer. However, a change that does not impact behavior at a user-level will not translate into impact at a group-level. So you may waste time waiting for results from a group-level test that has no impact.
+
+In this case, it may be worth running a user-targeted experiment first in order to see if there is any impact on individual user behavior (which should be quicker to gain insights from because of higher statistical power) before running a group-targeted experiment.
+
+
+### 2. Gain user-level insights and identify randomization risks
+By running a user-targeted experiment first, you'll gain insight into how individual users interact with your change and what characteristics they may possess. This will then give you an understanding of what results you can expect from a group-targeted experiment.
+
+This can also help you identify randomization risks. For example, let's say a B2B SaaS app runs a user-targeted experiment and notices that users in large companies are more likely to interact with their new feature. In this case, when they run a group-targeted experiment, they can ensure to filter out small and medium-sized companies in their experiment in order to obtain more accurate results.
+
+
+## How to setup group-targeted experiments on PostHog
+
+First, ensure you have [setup a group](docs/product-analytics/group-analytics#getting-started) in PostHog. Next, go to your PostHog dashboard and create a [new experiment](docs/experiments/manual#creating-an-experiment).
+
+In the New Experiment screen, you'll be able to select the Participant type. By default, it will show `Persons` (i.e. a user-level experiment). Click on the drop-down and select your new group.
+
+![Screenshot of setting the participant type in an experiment](../images/tutorials/when-and-how-to-run-group-targeted-a-b-tests/participant-type.png)
+
+Then set your experiment goal and secondary metrics as usual. A handy feature that's especially useful for group-level experiments is the Minimum Acceptable Improvement calculator at the bottom of the page. Since group-level experiments have lower statistical power, this will recommend a time for you for how long you should run your experiment in order to see results.
+
+![Screenshot of the Minimum Acceptable Improvement calculator](../images/tutorials/when-and-how-to-run-group-targeted-a-b-tests/minimum-acceptable-improvement.png)
+
```

Comment:
There are some more steps missing here:
1. Click create experiment
2. Implement, add flags to code
3. Launch, wait for results
4. How to understand if your experiment is relevant, had an impact, and avoided risks (this ties this section to the rest of the piece)

Reply:
Hmm Im not sure we need to go into that much detail. The specifics of how to create and run an experiment as usual are hyperlinked above